sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_delete_events_v2/query.sql

```diff
@@ -25,6 +25,10 @@ WHERE
     AND DATE_ADD(@submission_date, INTERVAL 1 DAY)
   )
   AND DATE(`timestamp`) = @submission_date
-  AND jsonPayload.type = 'activityEvent'
-  AND jsonPayload.fields.event = 'account.deleted'
+  AND (
+    jsonPayload.type IN ('accountDeleted', 'accountDeleted.byRequest', 'DB.deleteAccount')
+    OR (jsonPayload.type = 'activityEvent' AND jsonPayload.fields.event = 'account.deleted')
+  )
   AND jsonPayload.fields.uid IS NOT NULL
+GROUP BY
+  jsonPayload.fields.uid
```

Comment:
Should we group by `hmac_user_id` instead?

Reply:
While we could instead group by the `user_id` and `hmac_user_id` result columns, it's simpler and more efficient to just group by the `jsonPayload.fields.uid` input column since `user_id` and `hmac_user_id` are directly calculated from `jsonPayload.fields.uid`.